content/api/hub/plugin_spin_otel.md

```diff
@@ -0,0 +1,95 @@
+title = "Spin OTel"
+template = "render_hub_content_body"
+date = "2024-08-16T00:22:56Z"
+content-type = "text/plain"
+tags = ["otel", "spin", "traces", "metrics", "logs", "observability"]
```

Comment:
I would limit it to 3 tags. The rest can be added as search terms using the `keywords` filed under the `[extra]` section.

ref: https://github.com/fermyon/developer/blob/a34891d88acb7c1bc519fdfdd3267d7374838211/content/api/hub/plugin_js2wasm.md?plain=1#L17

Reply:
Also `spin` is not a very useful tag in this context, because, well _gestures_